### ISSUE TITLE
Default can be set in global settings

### ISSUE THREAD

**Issue by Shnoulle** [TonisOrmisson/limesurvey-url-templates#2]
It's just a simple idea (uneeded currently for me) : get the [default](https://github.com/TonisOrmisson/limesurvey-url-templates/blob/d6f17e0369e086e5e42c25ce21996e55948b5919/UrlTemplateChanger.php#L52)  by a plugin global settings :).

Then if user update global settings with removing all json : no default, this is deactivated everywhere.

And the [params](https://github.com/TonisOrmisson/limesurvey-url-templates/blob/d6f17e0369e086e5e42c25ce21996e55948b5919/UrlTemplateChanger.php#L69) too :)

**Comment by TonisOrmisson** [comment]
Yes. A good one. I am also currently not actively using this. But I will pick this up some time in the future.

And you are always welcome to PR this :)
